cabal-install/Distribution/Client/Dependency.hs

```diff
@@ -152,7 +154,8 @@ data DepResolverParams = DepResolverParams {
        depResolverAvoidReinstalls   :: Bool,
        depResolverShadowPkgs        :: Bool,
        depResolverStrongFlags       :: Bool,
-       depResolverMaxBackjumps      :: Maybe Int
+       depResolverMaxBackjumps      :: Maybe Int,
+       depResolverEnableBackjumping :: EnableBackjumping
```

Comment:
That's right.  Maybe I should rename the option, because it doesn't disable the backjump limit.  I used the backjump limit to limit the run time and memory usage of the individual test cases.  The backjump messages in the solver's log are also necessary because they are used for truncating the summarized log.  They prevent the solver from keeping the whole log in memory.